src/app/components/universal-tabbed-card/universal-tabbed-card.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="container card">
+  <ul class="nav nav-pills" id="pills-tab" role="tablist">
+    <li class="nav-item" role="presentation" *ngFor="let tab of childTabs; let i = index">
+      <button 
+        class="nav-link" 
+        [ngClass]="(i === activeTab ? 'active' : '')" 
+        [id]="'pills-'+ i +'-'+ uniqueId +'-tab'"
+        data-bs-toggle="pill" 
+        [attr.data-bs-target]="'#pills-'+ i +'-'+ uniqueId"
+        type="button"
+        role="tab"
+        [attr.aria-controls]="'pills-'+ i +'-'+ uniqueId"
+        [attr.aria-selected]="(i === activeTab ? 'active' : '')"
+      >{{ tabTitles[i] }}</button>
+    </li>
+  </ul>
+  <div class="tab-content">
+    <ng-content></ng-content>
```

Comment:
youll have to learn me about `<ng-content>`  it may be used in the udemy, but im not super familiar since we dont use it in atlas

Reply:
ng-content is basically a placeholder for HTML that allows you to throw HTML into the component depending on what's needed. If you check the services-offered-component it calls <app-universal-tabbed-card> and then adds tabs and HTML between the opening and closing tags.